src/libraries/System.Net.Security/tests/FunctionalTests/TestConfiguration.cs

```diff
@@ -38,7 +38,8 @@ public static Task WhenAllOrAnyFailedWithTimeout(params Task[] tasks)
             // On Windows, null ciphers (no encryption) are supported.
             if (OperatingSystem.IsWindows())
             {
-                return true;
+                // This may be more complicated but Server 2022 and Windows 11 some with restricted set of default ciphers.
```

Comment:
?
```suggestion
                // This may be more complicated but Server 2022 and Windows 11 are some with restricted set of default ciphers.
```